docs/sources/get-started/quick-start.md

```diff
@@ -194,165 +192,337 @@ You can view your logs using the command line interface, [LogCLI](/docs/loki/<LO
+- `level=INFO` is the log level.
+- `line=97` is the line number in the code.
+- `msg="192.168.65.1 - - [21/Feb/2025 16:09:42] "GET /static/style.css HTTP/1.1" 304 -"` is the log message.
+
+<!-- INTERACTIVE copy END -->
+When querying Loki, you can pipe the result of the label selector through a parser. This extracts attributes from the log line for further processing. For example, lets pipe `{container="greenhouse-main_app-1"}` through the `logfmt` parser to extract the `level` and `line` attributes:
+<!-- INTERACTIVE copy START -->
+```logql
+{container="greenhouse-main_app-1"} | logfmt
 ```
 <!-- INTERACTIVE copy END -->
-The flog app generates log lines for simulated HTTP requests.
+When you now expand a log line in the query result, you will see the extracted attributes.
+
+{{< admonition type="tip" >}}
+Before we move on to the next section, let's generate some error logs. To do this, enable the bug service in the sample application. This is done by setting the `Toggle Error Mode` to `On` in the Carnivorous Greenhouse application. This will cause the bug service to randomly cause services to fail.
+{{< /admonition >}}
+
+### Advanced and Metrics Queries
 
-To see all `GET` log lines, enter the LogQL query:
+With Error Mode enabled the bug service will start causing services to fail, in these next few LogQL examples we will track down some of these errors.  Lets start by parsing the logs to extract the `level` attribute and then filter for logs with a `level` of `ERROR`:
 <!-- INTERACTIVE copy START -->
-```bash
-{container="evaluate-loki-flog-1"} |= "GET"
+```logql
+{container="greenhouse-plant_service-1"} | logfmt | level="ERROR"
 ```
 <!-- INTERACTIVE copy END -->
-To see all `POST` methods, enter the LogQL query:
+This query will return all the logs from the `greenhouse-plant_service-1` container that have a `level` attribute of `ERROR`. You can further refine this query by filtering for a specific code line:
 <!-- INTERACTIVE copy START -->
-```bash
-{container="evaluate-loki-flog-1"} |= "POST"
+```logql
+{container="greenhouse-plant_service-1"} | logfmt | level="ERROR", line="58"
 ```
 <!-- INTERACTIVE copy END -->
-To see every log line with a 401 status (unauthorized error), enter the LogQL query:
+This query will return all the logs from the `greenhouse-plant_service-1` container that have a `level` attribute of `ERROR` and a `line` attribute of `58`.
+
+LogQL also supports metrics queries. Metrics are useful for abstracting the raw log data aggregating attributes into numeric values. This allows you to utilise more visualization options in Grafana as well as generate alerts on your logs. 
+
+For example, you can use a metric query to count the number of logs per second that have a specific attribute:
 <!-- INTERACTIVE copy START -->
-```bash
-{container="evaluate-loki-flog-1"} | json | status="401"
+```logql
+sum(rate({container="greenhouse-plant_service-1"} | logfmt | level="ERROR" [$__auto]))
```

Comment:
The data from this query is hard to see in the graph when it's set to `lines` since it's just a couple of green dots.  It might be helpful to add another sentence here and direct users to change the graph view to `bars` to make the data more visible in the graph.  That would also teach new users that there are multiple graph visualizations available that they can play around with.